client-common/src/storage/memory_storage.rs

```diff
@@ -151,3 +151,43 @@ impl Storage for MemoryStorage {
         Ok(keyspaces)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use blake2::Blake2s;
+    use chain_core::common::hash256;
+
+    #[test]
+    fn check_memorystorage_ordering() {
```

Comment:
this doesn't check the order against sled or what the problem was?

Reply:
it was in memory storage only, sled has no problem.
we didn't use keys() for ordering previously, there was no issue before.